### PR TITLE
elementary OS Cheat Sheet

### PR DESCRIPTION
**Pull request by namandistro**
What does your Instant Answer do?

- Shows all The Elementary OS Commands as cheat sheet.

What is the data source for your Instant Answer? (Provide a link if possible)

- http://lmelinux.net/2014/07/26/elementary-os-keyboard-shortcuts/

Why did you choose this data source?

- One of the Best and even recommended.

Are there any other alternative (better) data sources?

- There are chances.

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

- Elementary OS users, Geeks, Linux.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

- Yes, https://duck.co/ideas/idea/6240/elementary-os-keyboard-shortcuts

Are you having any problems? Do you need our help with anything?

- Nopes.

screenshot from 2015-08-14 19 13 09

-----
IA Page: https://duck.co/ia/view/elementary_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@namandistro thanks for contributing! We'll give you some feedback shortly.

BTW, don't forget to post a screenshot in the PR!

**Comment by namandistro** [comment]
@Moollaza I will update it tomorrow, I just made this cheat sheet while teaching other how community, ideas and contribution works in ddg its too small cheat sheet

**Comment by namandistro** [comment]
@moollaza Just have a look

**Comment by aleksandar-todorovic** [comment]
@namandistro As a regular contributor to elementary's community, I wanted to warn you that the word `elementary` in `elementary OS` should be spelled with a lowercase letter `e`. Please, make a quick fix before merging this pull request.

Sources:
* http://blog.elementary.io/post/107640994166/the-importance-of-our-brand (Internal Branding Guidelines section)
* http://elementaryos.stackexchange.com/questions/850/how-should-i-type-elementary-os-with-a-lowercase-e-or-an-uppercase-e/851#851
* http://meta.elementaryos.stackexchange.com/questions/5/lets-enforce-branding-guidelines-in-edits

**Comment by aleksandar-todorovic** [comment]
After further examining the list, I found lots of keyboard shortcuts that weren't included in that article:

* ⌘ + 1-9 will switch to a specific workspace, ⌘ + Home will switch to the first workspace, ⌘ + End will switch to the last one.
* PrScr will... well, print the screen, but Ctrl + PrScr will save the screenshot to Clipboard, Shift + PrScr will allow you to select an area on the screen to be screenshoted, and Ctrl + Alt + PrScr will screenshot the specific window.
* ⌘ + T will start the Pantheon Terminal.
* Alt + Space will switch to the next keyboard layout.

I have no source for this, I just checked the default settings from my installation. But I could screenshot them if needed.

There's also a [list of keyboard shortcuts in Pantheon Terminal](http://elementaryos.stackexchange.com/questions/981/list-of-keyboard-shortcuts-for-terminal) which I think are worth mentioning.

File: share/goodie/cheat_sheets/elementary.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            {
+                "val":"Window Switcher",
+                "key":"[Alt] [Tab]"
+            },
+            {
+                "val":"Switch Windows Backwards",
+                "key":"[⇧] [Alt] [Tab]"
+            },
+            {
+                "val":"window overview",
```
Comment (aleksandar-todorovic) [inline share/goodie/cheat_sheets/elementary.json]: Shouldn't the first letters be capitalized like everywhere else in values?

**Comment by namandistro** [comment]
@aleksandar-todorovic thanks for the suggest I will change it as per recommend.

**Comment by mintsoft** [comment]
@aleksandar-todorovic thanks for the input!

**Comment by namandistro** [comment]
@aleksandar-todorovic Changed as per your recommended changes do have a look at it.....

**Comment by aleksandar-todorovic** [comment]
Looks good to me. :)

The only suggestion I could add is to follow the same pattern for capitalizing first words. For example, on line 30, the word "overview" is capitalized, while on line 26, it isn't.

File: share/goodie/cheat_sheets/elementary.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    "name":"elementary OS",
+    "metadata":{
+        "sourceName":"Lme Linux",
+        "sourceUrl":"http://lmelinux.net/2014/07/26/elementary-os-keyboard-shortcuts/"
+    },
+    "section_order":[
+        "General"
+    ],
+    "sections":{
+        "General":[
```
Comment (moollaza) [inline share/goodie/cheat_sheets/elementary.json]: Let's break this up into more specific sections. Please take a look at the Ubuntu, Windows, Mac and other cheat sheets to get a good idea of some ways these can be grouped.

File: share/goodie/cheat_sheets/elementary.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
+{
+    "id":"elementary_cheat_sheet",
+    "name":"elementary OS",
+    "metadata":{
+        "sourceName":"Lme Linux",
+        "sourceUrl":"http://lmelinux.net/2014/07/26/elementary-os-keyboard-shortcuts/"
+    },
+    "section_order":[
```
Comment (moollaza) [inline share/goodie/cheat_sheets/elementary.json]: We now have a new "aliases" property you can use to define additional triggers. I think we should add:

```json
"aliases": [ "elementary os" ]
```

**Comment by namandistro** [comment]
@moollaza check it out

**Comment by moollaza** [comment]
@namandistro congrats! It's live: https://duckduckgo.com/?q=elementary+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**